base/Dockerfile

```diff
@@ -28,7 +28,27 @@ RUN curl -sSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.ta
     curl -sSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz.sha256 -o /tmp/go.tar.gz.sha256 && \
     echo "$(cat /tmp/go.tar.gz.sha256)  /tmp/go.tar.gz" | sha256sum -c - && \
     tar -C /usr/local -vxzf /tmp/go.tar.gz && \
-    rm /tmp/go.tar.gz /tmp/go.tar.gz.sha256
+    rm /tmp/go.tar.gz /tmp/go.tar.gz.sha256 && \
+    # Remove unnecssary files from the Go distro.
```

Comment:
typo: unnecessary

Any rhyme or reason to how we can trim these in general?  Was this just manual inspection?